### PR TITLE
Membership Charter Draft

### PR DESCRIPTION
**Pull request by wordshaker**
## Description Of Change

Add a template of a membership charter for the membership committee to review

## Purpose Of Change

To make it clear what the expectations are for membership commitee membership.

## Related Issue
#87

### CODE REVIEW

**Comment by wordshaker** [comment]
Please comment with suggested edits.

**Comment by codehippie1** [comment]
@wordshaker Here are my thoughts on below two proposals.

1. Expected Level of Contribution
To remain a member of the committee we ask that you attempt to attend meetings at least once a month. If you are unable to attend at this frequency or have to miss a couple for extenuating circumstances, you will be expected to raise this with the committee. You won't need to provide specifics, it's to let the group know you intend to take part in future.
If you cannot attend at that frequency, you could also contribute by submitting issues or pull requests to the repository.

2. Termination of Membership
On a quarterly basis, the chair, scribe or appointed representative will review levels of contribution from [members of the membership GitHub Teams](https://github.com/orgs/dotnet-foundation/teams/membership/members). If the expected contribution levels have not been met, with no reasoning, your membership to this team will be revoked.
After termination, you can re-apply to join the committee if you so wish.

Thoughts - Since these folks are volunteers, setting up dedicated timelines would deter new folks. For example, I started my journey with the .NET Foundation by accident. After getting tired of the Microsoft reactor at 11 Timesquare in NYC being shut down last year, I was looking for a space to host my .net meetups and saw a call for volunteers by Shawn Wildermuth on LinkedIn for the outreach committee. All I wanted to ask was where I could host my meetup, and I ended up working with some of the committees and some of the projects throughout 2022. 😊

Had I seen a similar membership and commitment requirement, I might not have joined the first call.

Suggestion 1 - If the membership committee sees no activity from a participant in a quarter, the committee can reach out to the member before the termination of the account. A notice before cancellation is a usual practice and might win some back.

Suggestion 2 - The term "member" is a loose term at the .NET Foundation. A "member" can be any of these.
- a member of the dotnet foundation
- a member project
- a member of any of the committees

Could we develop specific terms for the customers(member projects, foundation members) and providers(committee members)? Something similar to ASP.NET Insider/.NET Foundation Insider would help folks distinguish which side of the aisle a person is on.

**Comment by wordshaker** [comment]
> @wordshaker Here are my thoughts on below two proposals.
> 
> 1. Expected Level of Contribution
>    To remain a member of the committee we ask that you attempt to attend meetings at least once a month. If you are unable to attend at this frequency or have to miss a couple for extenuating circumstances, you will be expected to raise this with the committee. You won't need to provide specifics, it's to let the group know you intend to take part in future.
>    If you cannot attend at that frequency, you could also contribute by submitting issues or pull requests to the repository.
> 2. Termination of Membership
>    On a quarterly basis, the chair, scribe or appointed representative will review levels of contribution from [members of the membership GitHub Teams](https://github.com/orgs/dotnet-foundation/teams/membership/members). If the expected contribution levels have not been met, with no reasoning, your membership to this team will be revoked.
>    After termination, you can re-apply to join the committee if you so wish.
> 
> Thoughts - Since these folks are volunteers, setting up dedicated timelines would deter new folks. For example, I started my journey with the .NET Foundation by accident. After getting tired of the Microsoft reactor at 11 Timesquare in NYC being shut down last year, I was looking for a space to host my .net meetups and saw a call for volunteers by Shawn Wildermuth on LinkedIn for the outreach committee. All I wanted to ask was where I could host my meetup, and I ended up working with some of the committees and some of the projects throughout 2022. 😊
> 
> Had I seen a similar membership and commitment requirement, I might not have joined the first call.
> 
> Suggestion 1 - If the membership committee sees no activity from a participant in a quarter, the committee can reach out to the member before the termination of the account. A notice before cancellation is a usual practice and might win some back.
> 
> Suggestion 2 - The term "member" is a loose term at the .NET Foundation. A "member" can be any of these.
> 
> * a member of the dotnet foundation
> * a member project
> * a member of any of the committees
> 
> Could we develop specific terms for the customers(member projects, foundation members) and providers(committee members)? Something similar to ASP.NET Insider/.NET Foundation Insider would help folks distinguish which side of the aisle a person is on.

Hi @codehippie1,

Suggestion 1 - Totally understand that expected participation may be off putting and we could make it more lax. Notice before cancellation may be doable after the initial purge. There are currently 58 members of the teams account at at most 6 people who are active committee members. 

I'll make the edit to say we will reach out after not seeing activity for a quarter. We may have something different for the first clean down as I don't particularly want to manage comms with 58 individuals.

Suggestion 2 - I agree it's a pain; but it's the name of the name of the pre-established membership committee GitHub Teams. In the context of this document which is only about the Membership Committee I think it should be fine, I could maybe make the title more specific. I do agree that in general we need to sort out our common language though.

File: resources/commitee-documentation/membership-committee-charter.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+Minutes of each meeting are recorded, and can be viewed [here](../../meetings/).
+
+### Decisions and disputes
+Decisions are made via vote. If the committee cannot come to agreement on an action, it will be postponed or the chairperson will discuss with other board members to determine which decision would be in the best interests of the foundation.
+
+### Feedback to the board
+The Chairperson will feedback to the other members of the Board of Directors on a monthly basis. This includes topics that are currently in discussion, actions being taken and measures of success.
+
+## Duties, and Responsibilities
```
Comment (BillWagner) [inline resources/commitee-documentation/membership-committee-charter.md:64]: This section is almost verbatim the text in **Standard committee procedures**. I suggest merging them.

File: resources/commitee-documentation/membership-committee-charter.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Meetings have an agenda which is shared on GitHub and can be contributed to by any committee member. Each agenda item will have a GitHub Issue assigned which provides more context to the discussion point. 
+
+Minutes of each meeting are recorded, and can be viewed [here](../../meetings/).
+
+### Decisions and disputes
+Decisions are made via vote. If the committee cannot come to agreement on an action, it will be postponed or the chairperson will discuss with other board members to determine which decision would be in the best interests of the foundation.
+
+### Feedback to the board
+The Chairperson will feedback to the other members of the Board of Directors on a monthly basis. This includes topics that are currently in discussion, actions being taken and measures of success.
```
Comment (BillWagner) [inline resources/commitee-documentation/membership-committee-charter.md:62]: Should you emphasize that this is two way feedback? I found a lot of value in bringing board discussions to the membership committee for their thoughts.

File: resources/commitee-documentation/membership-committee-charter.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+To remain a member of the committee we ask that you attempt to attend meetings at least once a month. If you are unable to attend at this frequency or have to miss a couple for extenuating circumstances, you will be expected to raise this with the committee. You won't need to provide specifics, it's to let the group know you intend to take part in future.
+
+If you cannot attend at that frequency, you could also contribute by submitting issues or pull requests to the repository.
+
+Each committee meeting will have a draft agenda in GitHub available before the meeting. This is how to check when issues you are passionate about are going to be up for discussion and action votes.
+
+## Termination of Membership
+
+On a quarterly basis, the chair, scribe or appointed representative will review levels of contribution from [members of the membership GitHub Teams](https://github.com/orgs/dotnet-foundation/teams/membership/members). If the expected contribution levels have not been met, with no reasoning, your membership to this team will be revoked.
```
Comment (BillWagner) [inline resources/commitee-documentation/membership-committee-charter.md:39]: I like this. It's important that the list of committee members accurately reflect who is actively participating. I liked your idea on another thread to provide some notice when this happens.

File: resources/commitee-documentation/membership-committee-charter.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+### Decisions and disputes
+Decisions are made via vote. If the committee cannot come to agreement on an action, it will be postponed or the chairperson will discuss with other board members to determine which decision would be in the best interests of the foundation.
+
+### Feedback to the board
+The Chairperson will feedback to the other members of the Board of Directors on a monthly basis. This includes topics that are currently in discussion, actions being taken and measures of success.
+
+## Duties, and Responsibilities
+
+The committee meets on the 1st and 3rd Wednesday of each month at 7:00pm UTC.
```
Comment (BillWagner) [inline resources/commitee-documentation/membership-committee-charter.md:66]: Is it worth revisiting the time periodically (maybe once a year?) That might make as committee members other commitments change, or as the makeup changes and different times work better in different time zones.

**Review by BillWagner — APPROVED** [review]
This looks great to me.

I had a couple thoughts for potential suggestions.

**Review by CalvinAllen — APPROVED** [review]

File: resources/commitee-documentation/membership-committee-charter.md (hunk 1 above)
Comment (wordshaker) [reply resources/commitee-documentation/membership-committee-charter.md:57]: https://github.com/dotnet-foundation/wg-membership/pull/89/commits/5eaec58cf732e5e3133c2489703766568b062b5d

File: resources/commitee-documentation/membership-committee-charter.md (hunk 2 above)
Comment (wordshaker) [reply resources/commitee-documentation/membership-committee-charter.md:53]: https://github.com/dotnet-foundation/wg-membership/pull/89/commits/77ad480f6acbcae6e00e9ad395d4c2d2a7ee65c6

File: resources/commitee-documentation/membership-committee-charter.md (hunk 3 above)
Comment (wordshaker) [reply resources/commitee-documentation/membership-committee-charter.md]: https://github.com/dotnet-foundation/wg-membership/pull/89/commits/e738709f83df3f88f31328b0350e7731ceb549c7

File: resources/commitee-documentation/membership-committee-charter.md (hunk 4 above)
Comment (wordshaker) [reply resources/commitee-documentation/membership-committee-charter.md]: https://github.com/dotnet-foundation/wg-membership/pull/89/commits/78dd4cc5ff059750bf826be49d21d12dc7438540

File: resources/commitee-documentation/membership-committee-charter.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+The membership committee will also be informed and updated about board discussions, and their thoughts on these topics are encouraged.
+
+## Duties, and Responsibilities
+
+The committee meets on the 1st and 3rd Wednesday of each month at 7:00pm UTC. This time will be revisited once a year or as the makeup changes and different times work better in different time zones.
+
+Meetings have an agenda which is shared on GitHub and can be contributed to by any committee member. Each agenda item will have a GitHub Issue assigned which provides more context to the discussion point. 
+
+Minutes of each meeting are recorded, and can be viewed [here](../../meetings/).
```
Comment (wordshaker) [inline resources/commitee-documentation/membership-committee-charter.md:63]: Change phrasing to make clearer and fix link

**Comment by wordshaker** [comment]
I have addressed all the comments and will now merge. If there are any further amendments to be made, please raise a pull request 😄

**Review by 1137MOM — COMMENTED** [review]
GIOR

**Pull request merged**